sys/linux/dev_video4linux_amd64.const

```diff
@@ -441,7 +445,7 @@ V4L2_CID_ZOOM_CONTINUOUS = 10094863
 V4L2_CID_ZOOM_RELATIVE = 10094862
 V4L2_COLORSPACE_470_SYSTEM_BG = 6
 V4L2_COLORSPACE_470_SYSTEM_M = 5
-V4L2_COLORSPACE_ADOBERGB = 9
+# V4L2_COLORSPACE_ADOBERGB is not set
```

Comment:
If these are not present in the upstream kernel, we can as well start deleting these from descriptions. There is no point in keeping outdated descriptions for which we don't even have const values now.
Not necessary in this change. It would just be a useful cleanup as we start organizing this file.